src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -145,8 +145,8 @@ def _obtain_data_from_registry(login_server,
                 errorMessage = str(e)
                 logger.debug('Retrying %s with exception %s', i + 1, errorMessage)
                 time.sleep(retry_interval)
-    if errorMessage:
-        raise CLIError(errorMessage)
+        if errorMessage:
+            raise CLIError(errorMessage)
```

Comment:
Ugr, this is not a minor formatting update. This change the control flow. Please double check and make sure this is what you intended to do.

Reply:
This is an intended change, the error should be on the retry for loop.